src/sandstorm/sandstorm-http-bridge.c++

```diff
@@ -1358,53 +1374,26 @@ public:
     return true;
   }
 
-  class ApiRestorer: public capnp::SturdyRefRestorer<capnp::AnyPointer> {
-  public:
-    explicit ApiRestorer(SandstormApi<>::Client&& apiCap,
-                         capnp::Capability::Client&& sessionContext)
-        : apiCap(kj::mv(apiCap)), sessionContext(kj::mv(sessionContext)) {}
-
-    capnp::Capability::Client restore(capnp::AnyPointer::Reader ref) override {
-      // TODO(soon): As in Restorer::restore() (above), SandstormApi should be the "default cap"
-      //   exported on this connection.
-      // TODO(soon): We are exporting the SessionContext for the most-recently established session
-      //   under the name "HackSessionContext", but this is a hack. We should really be sending an
-      //   HTTP header to the app along with each request that gives a session ID which can be used
-      //   as part of a "Restore" command on the Cap'n Proto socket to get the *particular*
-      //   SessionContext associated with that request.
-      if (ref.isNull() || ref.getAs< ::capnp::Text>() == "SandstormApi") {
-        return apiCap;
-      } else if (ref.getAs< ::capnp::Text>() == "HackSessionContext") {
-        return sessionContext;
-      }
-
-      // TODO(someday):  Implement level 2 RPC?
-      KJ_FAIL_ASSERT("SturdyRefs not implemented.");
-    }
-
-  private:
-    SandstormApi<>::Client apiCap;
-    capnp::Capability::Client sessionContext;
-  };
-
   struct AcceptedConnection {
     kj::Own<kj::AsyncIoStream> connection;
     capnp::TwoPartyVatNetwork network;
     capnp::RpcSystem<capnp::rpc::twoparty::VatId> rpcSystem;
 
-    explicit AcceptedConnection(ApiRestorer& restorer, kj::Own<kj::AsyncIoStream>&& connectionParam)
-        : connection(kj::mv(connectionParam)),
-          network(*connection, capnp::rpc::twoparty::Side::SERVER),
-          rpcSystem(capnp::makeRpcServer(network, restorer)) {}
+    explicit AcceptedConnection(SandstormHttpBridge::Client& bridge,
+                                kj::Own<kj::AsyncIoStream>&& connectionParam)
+      : connection(kj::mv(connectionParam)),
+        network(*connection, capnp::rpc::twoparty::Side::SERVER),
+        rpcSystem(capnp::makeRpcServer(network, bridge)) {}
   };
 
-  kj::Promise<void> acceptLoop(kj::ConnectionReceiver& serverPort, ApiRestorer& restorer,
+  kj::Promise<void> acceptLoop(kj::ConnectionReceiver& serverPort,
+                               SandstormHttpBridge::Client& bridge,
```

Comment:
Oh, neat -- I was unaware that I could specify capture-by-move for a single variable and still specify capture-by-ref for the default. I think my main motivation here was to avoid the verbosity of writing out each captured variable.